crates/openapi/src/routes/payment_method.rs

```diff
@@ -0,0 +1,170 @@
+/// PaymentMethods - Create
+///
+///Creates and stores a payment method against a customer. In case of cards, this API should be used only by PCI compliant merchants
```

Comment:
```suggestion
/// Creates and stores a payment method against a customer. In case of cards, this API should be used only by PCI compliant merchants
```
Also split it if more than 100 characters.